.github/workflows/ci.yml

```diff
@@ -61,6 +61,7 @@ jobs:
     - uses: bufbuild/buf-action@v1
       with:
         setup_only: true
+    - uses: astral-sh/setup-uv@v6
```

Comment:
Can you pin this action? I've used https://github.com/suzuki-shunsuke/pinact to do so previously.

Reply:
did it with:

```console
$ pinact run -e "actions/.*" -e "bufbuild/.*"
```

since we trust `actions/` & `bufbuild/` actions.